### ISSUE TITLE
multiple uploads is broken

### ISSUE THREAD

**Issue by vandux** [lardawge/carrierwave_backgrounder#232]
related to issue #200 

my setup (active_job):

```
# property.rb

mount_uploaders :photos, PhotoUploader
store_in_background :photos, PhotoUploaderJob
```

fixing bug #200 leads you down a cascading path of errors

first #200 

```
NoMethodError (undefined method `cache_name' for #<CarrierWave::Mounter:0x007fbe443628b0>)
```

then after fixing am seeing:

```
NoMethodError (undefined method `remote_photos_url' for #<Property:0x007f4f9ec4b158> Did you mean? remote_photos_urls remote_photos_urls= remove_photos remove_photos? remove_photos= remove_photos!)

at lib/backgrounder/orm/activemodel.rb:24
```

after fixing this, am then seeing:

```
NoMethodError (undefined method `mounted_as' for #<Array:0x007f9e2ad382e8>): 
```

**Comment by lardawge** [comment]
This feature has never been officially released and is only in the master branch of carrierwave. Until it is released, or I need it personally, I will not put time into supporting it. Feel free to create a pull request.

**Comment by nowakov** [comment]
@shwing did you manage to somehow fix this? Can you share the code?
@lardawge Are you planning to update the gem? Do you need any help?

**Comment by shwing** [comment]
@nowakov no, I haven't moved the jobs to the background yet

**Comment by nowakov** [comment]
@shwing https://github.com/nowakov/carrierwave_backgrounder
I've fixed `mounted_as` issue and it works fine for me.

**Comment by lardawge** [comment]
Updating for those finding this via search.

Working towards an official 1.0 release in the 1.0-beta branch. This is a feature that isn't supported but will be added.

Full disclosure, I have no idea what the use case is for multiple files on a single column, however, I will try and support it. Should not be too difficult.

**Comment by lardawge** [comment]
Closing this. The feature is complete and will soon be merged into the master branch and released as 1.0.0-beta
